src/models/goal.ts

```diff
@@ -13,6 +13,16 @@ export enum GoalTypes {
   WORDS = 'words',
 }
 
+/**
+ * All possible durations of goals.
+ */
+export enum GoalDurations {
+  DAILY = 'daily',
+  MONTHLY = 'monthly',
+  WEEKLY = 'weekly',
+  YEARLY = 'yearly'
+}
```

Comment:
How are we converting these to a numerical duration?

Reply:
No idea yet. Will probably have to be a function because monthly can be 28, 29, 30, or 31

Reply:
That works.

Reply:
Did we ever decide whether monthly goals were going to be point-in-time or always end on the last day of the month?  Paging @jegillikin and his notes on our last meeting.

Reply:
Always at the last day of the month/year/whatever.